draft-ietf-snac-simple.xml

```diff
@@ -859,6 +859,66 @@
+	  reachability to the Internet or not.
+	</t>
+      </section>
+      <section>
+	<name>Use on properly managed network</name>
+	<t>
+	  In this scenario, a non-expert user attaches a stub router to an infrastructure network that's managed. This network has correctly deployed
+	  RA Guard and/or port-based access control. As a result, the stub router won't succeed in advertising a prefix on the managed network.
+	</t>
+	<t>
+	  In the managed network case, it's possible that the network operator is willing to permit stub routers to be attached to the network by
+	  users. In this case, they might either not deploy RA guard, or they might deploy working DHCPv6 prefix delegation. This could be
+	  PD-per-host (where hoses are encouraged to use prefix delegation) or just ordinary prefix delegation (where hosts are given prefix
+	  delegation if they ask for it, but not encouraged to ask for it).
+	</t>
+	<t>
+	  In such a situation, if mDNS works on the infrastructure link, the stub router will function correctly, because the delegated prefix
+	  will be correctly routed.
+	</t>
+	<t>
+	  It's worth noting that IPv4 devices that act similarly to stub routers, using NAT64, already exist and may indeed use the stub network
+	  functionality to support internal connections that aren't even apparent to the user. So it's not implausible that it might be desirable
+	  to enable such a use case for IPv6, and the current stub network document does in fact enable this use case.
+	</t>
+      </section>
+      <section>
+	<name>Use on a managed network without IPv6</name>
+	<t>
+	  A managed network without IPv6 is not expected to implement RA guard (although it probably should). On such a network, a stub router
+	  will connect and advertise services if they are not blocked by the infrastructure. A properly-behaved stub router should cause no
+	  operational problems, just as connecting an IPv4 NAT gateway in the same scenario would not cause operational problems.
+	</t>
+      </section>
+      <section>
+	<name>Use on an unmanaged (non-home) IPv6 network</name>
+	<t>
+	  If a stub router is connected to an unmanaged IPv6 network that does not implement RA guard, and the network provides only DHCPv6 numbering,
+	  the stub router will advertise an on-link prefix that permits autonomous configuration. Because the stub router is not advertising a
+	  default route, this should not cause any operational issues.
+	</t>
+	<t>
+	  It might be surprising to the network operator to see the stub router's prefix advertised on the link. However, a network
+	  operated by a knowledgable operator that does not implement RA guard can always find itself in this situation. This does
+	  not create any actual operational problem.
+	</t>
+	<t>
+	  We need to address the case where the network operator is not knowledgable and did not knowingly configure a unsuitable
```

Comment:
complex formulation: "did not knowingly configure a unsuitable prefix, ... "   - could this be simplified or split?

Was the operator not aware of the existence of SNAC? There can also be knowledgeable operators that are not aware that SNAC exists. Does it matter for the behavior here whether the operator knows a lot or knows a little? It's how the network was configured, that counts.

Reply:
The caption says "unmanaged" so why is there a network operator? (There isn't, it that case, right?)
Or is "badly managed" intended?

Reply:
A network always has an operator. The operator may or may not do any actual "management." In a typical home network, the "operator" is the home user. The home user is not expected to do any "management" of the network. Possibly this could be made clearer. :)

Reply:
Hmm, yes good to clarify this. The person who last plugged the Ethernet cable in or our may be the "operator" then. Such a typical operator isn't the type of person that would be surprised to see a particular prefix on the link because they don't look at tools showing prefixes and they don't know what these things are.   So that gave me a different impression of what an "operator" is here.